common/src/main/java/io/druid/concurrent/Execs.java

```diff
@@ -35,24 +37,52 @@
  */
 public class Execs
 {
-  public static ExecutorService singleThreaded(String nameFormat)
+  private static int nullToNormal(@Nullable Integer priority)
```

Comment:
the default is technically the priority of the creating thread.
Even if in our case this might be normal all the time, I think it will make things less cluttered to not not define a default everywhere and leave it null